modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ViewEntryUpperToolbar.es.js

```diff
@@ -28,6 +28,7 @@ function ViewEntryUpperToolbar({
 	additionalButtons,
 	children,
 	dataRecordId,
```

Comment:
Hey @jotascript, please rename this to `showNavigationControls` or something like this.

Thx